modules/FhirAPI/src/FhirAPI/FhirRestApiBuilder/Builders/EncounterBuilder.php

```diff
@@ -131,12 +131,13 @@ class	        token	        Classification of patient encounter	Encounter.class
         $this->createMapping(["FHIREncounter",'status',1,'EncounterStatus',"","form_encounter","form_encounter.status","EncounterStatus (Required)",self::EXACT,""]);
         $this->createMapping(["FHIREncounter",'service-type',1,'EncounterStatus',"","form_encounter","lr.option_id","EncounterStatus (Required)",self::EXACT,""]);
         $this->createMapping(["FHIREncounter",'service-provider',1,'EncounterStatus',"","form_encounter","facility_id","EncounterStatus (Required)",self::EXACT,""]);
-
         $this->createMapping(["FHIREncounter",'arrival-way',1,'form_encounter',"","form_encounter","arrival_way"," ",self::PART,""]);
         $this->createMapping(["FHIREncounter",'reason-codes-details',1,'form_encounter',"","form_encounter","reason_codes_details"," ",self::PART,""]);
 
+        $this->createMapping(["FHIREncounter",'status-extended',1,'form_encounter',"","form_encounter","all_statuses"," ",self::EXACT,""]);
+
         /////////////////////////////////////////////SORT BY id,date,appointment,patient,status    from mysql table form_encounter column id
-        $this->createMapping(["FHIREncounter",'_sort',1,'token',"id,date,appointment,patient,status,priority,service-type","form_encounter","form_encounter.id,form_encounter.date,form_encounter.eid,form_encounter.pid,form_encounter.status,form_encounter.priority,service_type_seq",'A code for the type of Encounter',self::EXACT,""]);
+        $this->createMapping(["FHIREncounter",'_sort',1,'token',"id,date,appointment,patient,status,priority,service-type,status_update_date","form_encounter","form_encounter.id,form_encounter.date,form_encounter.eid,form_encounter.pid,form_encounter.status,form_encounter.priority,service_type_seq,form_encounter.status_update_date",'A code for the type of Encounter',self::EXACT,""]);
```

Comment:
Add new search parameter for status-update-date (with all the modifiers of date), we need it for current development

Reply:
ok but note that
all of this config needs to go away ....
This is garbage code that no one will understand in the future